MIGRATION.md

```diff
@@ -628,6 +629,14 @@ In 7.0 we've removed the following feature flags:
 | `breakingChangesV7` | This flag is no longer needed and should be deleted.        |
 | `babelModeV7`       | See [Babel mode v7 exclusively](#babel-mode-v7-exclusively) |
 
+#### Dark mode detection
+
+Storybook 7 uses `prefers-color-scheme` to detects your system's dark mode preference if a theme is not set.
+
+Earlier versions used the light theme by default, so if you don't set a theme and your system's settings are in dark mode, this could surprise you.
+
+To manually set a theme, see [the theming docs](https://storybook.js.org/docs/react/configure/theming).
```

Comment:
A couple of tweaks here @shilman that we can use if you don't mind, we can probably adjust this to the following:
```md
Storybook 7 uses [`prefers-colour-scheme`](https://developer.mozilla.org/en-US/docs/Web/CSS/@media/prefers-color-scheme) to detect your system's dark mode preference if a theme is not set.

Earlier versions used the light theme by default, so if you don't set a theme and your system's settings are in dark mode, this could surprise you.

To learn more about theming, read our [documentation](https://storybook.js.org/docs/react/configure/theming).
```

Reply:
I applied this change for you @shilman